scripts/csv_importer.py

```diff
@@ -0,0 +1,398 @@
+#!/usr/bin/env python3
+
+import argparse
+import ast
+import csv
+import datetime
+import json
+import subprocess
+import sys
+import uuid
+import yaml
+
+try:
+    import mysql.connector
+    from mysql.connector.connection import MySQLCursor
+except ImportError:
+    print("Module mysql.connector is required.")
+    print("You can install it with: pip install mysql-connector-python")
+    sys.exit(1)
+
+
+DEFAULT_COLLECTION_VISIBILITY = "private"  # can be changed to "public" or "internal"
+
+
+def get_current_time() -> str:
+    """Returns the current time in YYYY-MM-DD-hh:mm:ss format"""
+    return datetime.datetime.now().strftime('%Y-%m-%d %H:%M:%S')
+
+
+def get_database_host(container_name: str) -> str:
+    """
+    Tries to automatically find the database host ip by container name.
+    :param container_name: name of the container to find ip address of
+    :return: ip address of the container
+    """
+    host = subprocess.run(
+        "sudo docker inspect -f '{{range.NetworkSettings.Networks}}{{.IPAddress}}{{end}}' " + container_name,
+        shell=True, check=True, capture_output=True, encoding="utf-8"
+    ).stdout
+
+    if not host:
```

Comment:
I propose exchange `not host` into 
```python
try:
    ipaddress.ip_address(host)
catch ValueError as exc
    #invalid address print()
```

https://docs.python.org/3/library/ipaddress.html#convenience-factory-functions
Please consider